00-HTML-CSS-basics/html excercises/index.css

```diff
@@ -0,0 +1,66 @@
+body {
```

Comment:
No encuentro las respuestas a:

- How could you add weight to the global font definition to win over the classes added by point 3?
- Imagine there is a declaration like class=”oh-no-inline-styles” style=”background:red” and you need to change the background to green without changing the inline style. How could you accomplish this?

Si estan dentro del archivo y se me pasaron podes marcarlas con un comment porfa.

- Falto aplicar responsive a este diseño.

Además un comentario muy menor: No se respeta el nombre de archivo sugerido

Reply:
Agregué un .txt con mis conclusiones a esas preguntas en un commit